configure.ac

```diff
@@ -930,6 +930,15 @@ AS_CASE(["$target_cpu"], [[i[3-6]86*]], [
 ])
 
 OPT_DIR=
+AC_ARG_WITH([gmp-dir],
+  AS_HELP_STRING([--with-gmp-dir=DIR],
+                 [specify the prefix directory where gmp is installed]),
+  [OPT_DIR="$withval"], [])
+AC_ARG_WITH([gmp],
+  [AS_HELP_STRING([--without-gmp],
+                  [disable GNU GMP to accelerate Bignum operations])],
+  [OPT_DIR=], [with_gmp=yes])
```

Comment:
This clears `$with_gmp_dir` added to `OPT_DIR` above, when `configure --with-gmp --with-gmp-dir=/path/to/gmp`.